content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md

```diff
@@ -58,20 +54,22 @@ The [latest version of the .NET Tracer][4] can automatically instrument the foll
 | ASP.NET MVC                     | `Microsoft.AspNet.Mvc` 4.0+                                                               | `AspNetMvc`          |
```

Comment:
Looks like the ADO.NET line above also needs updating like in the dotnet-core.md file

Reply:
@sarina-dd or @apigirl can you replace the ADO.NET line above with the following text I copied from the `content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md` file:

```
| ADO.NET | All AdoNet integrations | `AdoNet` |
```

Reply:
@zacharycmontoya Only other note here is for NuGet package we have `All AdoNet integrations`. Do we mean each integration has an individual NuGet package? Might change the verbiage a bit if that's the case to avoid confusion as all the others either have built-in or the package listed.

Reply:
So sorry, I didn't see your comment until now. This integration is unique because instead of toggling a specific integration, it toggles multiple integrations (e.g. `SQL Server`, `MySql`, `Oracle`, etc.). Maybe this could be reworded to say `All supported AdoNet integrations`. I suggest we either make this specific change or we can reword it in a separate PR so we can get the rest of the docs out there since the team already approved the previous `All AdoNet integrations` wording

Reply:
Sounds good, we can leave as-is for now. Thanks for the information! Kaylyn already approved, so whenever ya'll are ready to merge, just drop us a line in the Docs channel and we can merge for you.